content/en/guides/configuration-glossary/configuration-router.md

```diff
@@ -19,6 +19,8 @@ This can be useful if you need to serve Nuxt as a different context root, from w
 
 If you want to have a redirect to `router.base`, you can do so [using a Hook, see _Redirect to router.base when not on root_](/docs/2.x/configuration-glossary/configuration-hooks#redirect-to-routerbase-when-not-on-root).
 
+In Nuxt 2.15+, this property can be set using environment variables to change the configuration of an app that has already been built.
```

Comment:
Can we specify the environnement variable name?

Reply:
Maybe an example would be helpful? Something like:
```js
export default {
  router: {
    base: process.env.MY_BASE_URL || '/',
  }
}
```